commands/new_function.go

```diff
@@ -123,91 +123,97 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("unable to find file: %s - %s", appendFile, statErr.Error())
 		}
 
-		duplicateError := duplicateFunctionName(functionName, appendFile)
+		var duplicateError error
+		services, duplicateError = duplicateFunctionName(functionName, appendFile)
```

Comment:
@alexellis would you like me to add an additional commit here to cover ensuring past functions get `latest` tag if they aren't tagged? I was going to add this, but I thought the validation could possibly be done elsewhere and hence left for a potential debate. Just let me know. 😊👍